vowpalwabbit/parse_args.cc

```diff
@@ -1802,7 +1803,8 @@ VW::workspace* initialize_with_builder(int argc, char* argv[], io_buf* model, bo
     trace_message_t trace_listener, void* trace_context, std::unique_ptr<VW::setup_base_i> learner_builder)
 {
   std::unique_ptr<options_i, options_deleter_type> options(
-      new config::options_boost_po(argc, argv), [](VW::config::options_i* ptr) { delete ptr; });
+      new config::options_cli(std::vector<std::string>(argv + 1, argv + argc)),
```

Comment:
can we use vw's make unique here?

Reply:
You can't use a deleter with `std::make_unique` or `VW::make_unique` unfortunately